src/catkin_pkg/cmake.py

```diff
@@ -38,33 +38,33 @@
 
 
 def get_metapackage_cmake_template_path():
-    '''
-    Returns the location of the metapackage CMakeLists.txt CMake template.
+    """
+    Return the location of the metapackage CMakeLists.txt CMake template.
 
     :returns: ``str`` location of the metapackage CMakeLists.txt CMake template
-    '''
+    """
     rel_path = os.path.join('templates', 'metapackage.cmake.in')
     return os.path.join(os.path.dirname(__file__), rel_path)
 
 
 def configure_file(template_file, environment):
-    '''
-    Evaluate a .in template file used in CMake with configure_file().
+    """
+    Evaluate a .in template file used in CMake with configure_string().
```

Comment:
Can you please explain why you change this?

Reply:
The original comment triggered `D402 First line should not be the function's "signature"` issue.

The description  does not make sense to me - _`configure_file()` is doing `Evaluate .. with configure_file()`_.

Here is the function body:
```
    with open(template_file, 'r') as f:
        template = f.read()
        return configure_string(template, environment)
```

I assumed that there is a typo in the original comment, and replaced `configure_file()` with `configure_string()`.

The alternative is to revert the change and to suppress D402.

Reply:
This Python function has the same functionality as CMakes `configure_file` function. So the comment should stay as is and the warning can be suppressed.

Reply:
Fixed. 
Note - the suppression says 'Ignore only D402'; in reality, all issues in the function definition and the comment are suppressed. I was not able to find a way to suppress the issue for the comment only.